test/system/030-run.bats

```diff
@@ -870,4 +870,19 @@ EOF
     run_podman container rm -fa
 }
 
+@test "podman run failed --rm " {
+    rand=$(random_string 30)
+    run_podman run -p 1200:80 --rm -d --name $rand docker.io/library/nginx
```

Comment:
I thought we actually needed a process to bind, but it looks like we can do it without.  Besides this is the image (sortof) that the user reported.  Since $IMAGE will cause the same error, I will change.

Reply:
The bind is done by podman on the host, it will fail with all images.